056_digitsToWords.js

```diff
@@ -0,0 +1,43 @@
+digitsWords = (num) => {
+  let numWord = {
+    "0": "Zero",
+    "1": "One",
+    "2": "Two",
+    "3": "Three",
+    "4": "Four",
+    "5": "Five",
+    "6": "Six",
+    "7": "Seven",
+    "8": "Eight",
+    "9": "Nine",
+  };
+
+  numString = "";
+
+  for (let i = 0; i < num.toString().length; i++) {
```

Comment:
You are calling num.toString() in the loop. You can save it before entering the loop and use  it